app/editor/src/components/tab-control/hooks/useTab.tsx

```diff
@@ -33,7 +33,7 @@ export const useTab = (props?: ITabProps) => {
   const navigateTab = React.useCallback(
     (id: number, path: string = '/contents', open?: NavigateOptions) => {
       const url = `${path}/${id}${values?.showNav ? `?showNav=${values.showNav}` : ''}`;
-      const nav = open ?? values?.open;
+      const nav = open ?? values?.open ?? NavigateOptions.NewTab;
```

Comment:
The first time a user opens the app it will now default to NewTab instead of TwoTabView.